wmi_check/README.md

```diff
@@ -198,7 +198,7 @@ Need help? Contact [Datadog support][13].
 [3]: https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.performancecounter
 [4]: https://wmie.codeplex.com
 [5]: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-wmiobject
-[6]: https://docs.datadoghq.com/integrations/faq/how-to-retrieve-wmi-metrics/
+[6]: https://docs.datadoghq.com/integrations/guide/retrieving-wmi-metrics/
```

Comment:
Minor, but maybe also adjust the link text to match the page title?

Reply:
Thanks for catching this, updated!